lafleur/corpus_manager.py

```diff
@@ -32,6 +33,18 @@
 )
 
 
+def parse_args():
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(description="Fuzzing corpus manager")
+    parser.add_argument(
+        "--timeout",
+        type=int,
+        default=10,
+        help="Timeout in seconds for script execution (default: 10)"
+    )
+    return parser.parse_args()
+
+
```

Comment:
```suggestion
```
We don't actually parse the timeout in this file, see below for how it should be passed.